<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>322</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Julius Daemon</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="juliusd.qrc" >:/images/juliustray.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbStart" >
        <property name="text" >
         <string>Server &amp;starten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbReload" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Server &amp;neu starten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbStop" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Server sto&amp;ppen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="twLogs" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Server</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QTextEdit" name="tbLog" >
          <property name="undoRedoEnabled" >
           <bool>false</bool>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbSendWord" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Senden eines &amp;Testwortes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>&amp;Beenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei" >
    <property name="title" >
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionReload" />
    <addaction name="actionStop" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuEinstellungen" >
    <property name="title" >
     <string>&amp;Einstellungen</string>
    </property>
    <addaction name="actionSettings" />
    <addaction name="actionUsers" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
    <addaction name="actionSimon" />
    <addaction name="actionQt" />
   </widget>
   <addaction name="menuDatei" />
   <addaction name="menuEinstellungen" />
   <addaction name="menuAbout" />
  </widget>
  <action name="actionClose" >
   <property name="text" >
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="text" >
    <string>Server &amp;starten</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="text" >
    <string>Server &amp;neu starten</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Server sto&amp;ppen</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="text" >
    <string>&amp;Konfiguration</string>
   </property>
  </action>
  <action name="actionSimon" >
   <property name="text" >
    <string>simon</string>
   </property>
  </action>
  <action name="actionQt" >
   <property name="text" >
    <string>Qt</string>
   </property>
  </action>
  <action name="actionUsers" >
   <property name="text" >
    <string>&amp;Benutzer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="juliusd.qrc" />
 </resources>
 <connections/>
</ui>
